tests/utils/git_test.py

```diff
@@ -17,88 +17,76 @@
 
 import logging
 import os
-import shutil
 from unittest import mock
 
 import pytest
+from git import Repo
 
 from esrally import exceptions
-from esrally.utils import git, process
+from esrally.utils import git
+
+
+def commit(repo, *, date=None):
+    file_name = os.path.join(repo.working_dir, "test")
+    with open(file_name, "wb"):
+        pass
+    repo.index.add([file_name])
+    repo.index.commit("initial commit", commit_date=date)
 
 
 @pytest.fixture(scope="class", autouse=True)
 def setup(request, tmp_path_factory):
-    request.cls.remote_repo = "esrally"
-    request.cls.local_branch = "rally-unit-test-local-only-branch"
-    request.cls.remote_branch = "rally-unit-test-remote-only-branch"
-    request.cls.rebase_branch = "rally-unit-test-rebase-branch"
+    cls = request.cls
+
+    cls.local_remote = "remote_repo"
```

Comment:
Nit, but I found the name confusing.
```suggestion
    cls.local_remote_name = "remote_repo"
```